conans/client/rest/auth_manager.py

```diff
@@ -224,5 +224,32 @@ def authenticate(self, user, password):
             raise ConanException("Password contains not allowed symbols")
 
         # Store result in DB
-        remote_name, prev_user, user = update_localdb(self._localdb, user, token, self._remote)
+        remote_name, prev_user, user = self._update_localdb(user, token, self._remote)
         return token, remote_name, prev_user, user
+
+    def users_list(self, remotes):
```

Comment:
The ``auth_manager`` main responsibility is to handle authentication in the remote server. Don't put this functionality here if it is not authenticating in the remotes.